ci-operator/step-registry/openshift/powervc/podman/openshift-powervc-podman-commands.sh

```
@@ -0,0 +1,419 @@
+
+	#
+	podman run --rm -ti -v "${PWD}:/work:Z" -w /work \
+		"quay.io/hamzy/pvcctl:${PVCCTL_VERSION}" image import --gen-config
+
+	if [ ! -f config.yaml ]; then
+		log_error "config.yaml missing after --gen-config"
+		exit 1
+	fi
+
+	#
+	yq --inplace '.svc.host = "'$(cat "${SECRETS_DIR}/SVC_HOST")'"' config.yaml
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)